src/make.js

```diff
@@ -0,0 +1,19 @@
+const sum = function(accumulator, current) {
+	return accumulator + current;
+}
+
+const make = function(...argsFirst) {
+	let arr = [];
```

Comment:
**let** array save link to [], this link is't changed. in this case need const declaration